qui/qubes-updater-base.css

```diff
@@ -0,0 +1,227 @@
+.ok_button {
+    font-family: 'Source Sans Pro';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
```

Comment:
Use percentage values, not hardcoded sizes, because otherwise people with nonstandard DPIs/resolutions will be unhappy.